### ISSUE TITLE
Please add the ability for http and https urls in descriptions to be clickable and open in a web browser

### ISSUE THREAD

**Issue by bp88** [erikberglund/ProfileCreator#95]
Currently if a profile manifest has a `pfm_description` which contains a string with a URL, you cannot click on it. This would make it easier to reference the information that the description might be pointing you to.

**Comment by erikberglund** [comment]
Yes, that is the plan. There are a few fields that currently are populated and maintained in the manifests that doesn't have a GUI outlet yet. I haven't had the time to add this yet, but it will be included in the future!
